### DISCUSSION TITLE
Texture + Diffuse Gradient?

### DISCUSSION THREAD

**Discussion by EnrichoUser** [bnpr/Malt#358] [Q&A] [answered]
Good day, I was wondering on how one would have a transparent texture that can react to lighting.

![Q1](https://user-images.githubusercontent.com/108502891/179734755-dad371a7-19b7-4ced-8fd1-754ba5280c2c.png)

I have this rose here that I would like to be affected by a diffuse gradient shader, whilst retaining it's appearance.

![Q2](https://user-images.githubusercontent.com/108502891/179734771-bfeabf38-f7cb-4bbd-9507-3e0b23a8ac2a.png)

I tried this combination of nodes which does the job of affecting only the visible parts by lighting; but it inherits the full color ( The appearance/color of the texture becomes one) of the diffuse gradient. I am wondering if there's a way for it to retain it's appearance whilst also changing its color in response to light.

**Comment by pragma37** [comment, marked as answer]
You just have to multiply the texture color with the diffuse color.

**Reply by EnrichoUser** [reply]
Thank you.
